src/test/java/domain/LottoMachineTest.java

```diff
@@ -3,15 +3,29 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import java.util.Arrays;
+
+import static domain.LottoTest.createLottoNumbers;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LottoMachineTest {
 
     @DisplayName("로또를 자동으로 발행한다")
     @Test
-    void createLotto() {
-        assertThat(new LottoMachine(new Payment(2000))
-                .createAutoLottos())
-                .isInstanceOf(Lottos.class);
+    void create_auto_lottos() {
+        assertThat(new LottoMachine()
+                .createAutoLottos(3)
+                .getLottos()
+                .size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(new LottoMachine()
                .createAutoLottos(3)
                .getLottos()
                .size()).isEqualTo(3);
```
테스트 코드도 관리해야 하는 코드입니다.
new LottoMachine()
                .createAutoLottos(3)
                .getLottos()
                .size()
이 부분을 봣을때 알아보기가 힘든데요. 변수를 만들어서 가독성 있게 하면 어떨까요?